fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -1474,4 +1474,22 @@ public void testSearchAllergyIntoleranceWithWildcardMultipleIncludedAndProvenceR
         }
     }
 
-}
+    @Test(groups = { "server-search" }, dependsOnMethods = { "testCreateAllergyIntolerance", "testCreateProvenance" })
+    public void testSearchWithRelativePatientId() {
+        WebTarget target = getWebTarget();
+        Response response =
+                target.path("AllergyIntolerance").queryParam("patient", patientId).request(FHIRMediaType.APPLICATION_FHIR_JSON).header("X-FHIR-TENANT-ID", tenantName).header("X-FHIR-DSID", dataStoreId).get();
```

Comment:
minor formatting suggestion
```suggestion
        Response response = target.path("AllergyIntolerance")
                .queryParam("patient", patientId)
                .request(FHIRMediaType.APPLICATION_FHIR_JSON)
                .header("X-FHIR-TENANT-ID", tenantName)
                .header("X-FHIR-DSID", dataStoreId)
                .get();
```